.fixtures.yml

```diff
@@ -22,27 +24,40 @@ fixtures:
     augeasproviders_shellvar: https://github.com/simp/augeasproviders_shellvar
     augeasproviders_ssh: https://github.com/simp/augeasproviders_ssh
     augeasproviders_sysctl: https://github.com/simp/augeasproviders_sysctl
+    augeasproviders_syslog:
+      repo: https://github.com/hercules-team/augeasproviders_syslog
+      ref: 2.1.1
     autofs: https://github.com/simp/pupmod-simp-autofs
     chkrootkit: https://github.com/simp/pupmod-simp-chkrootkit
     clamav: https://github.com/simp/pupmod-simp-clamav
     concat:
       # master is at 4.0.1, but we are bound to < 4.0.0 in our metadata.json
       repo: https://github.com/simp/puppetlabs-concat
-      ref: 3.0.0
+      ref: 2.2.1
```

Comment:
We have moved to 3.0.0, this is not an accurate test

Reply:
@jeefberkey I noticed that, and then I noticed that 3.0.0 didn't resolve with many of the other fixtures (some augeasproviders IIRC constrained it to 2.x).  Sadly, we continue to use unconstrained  tracking branches in `.fixtures.yml`, and many of these repositories have recently made major releases that we _also_ don't support.

Unfortunately, the HEAD of simp-core's `Puppetfile.stable` hadn't been updated enough to  sort them out unambiguously, so I resolved what I could from this .fixtures file.  If you can suggest an alternate set of versioned tags that are more correct (and resolve), I'd certainly welcome a patch.